addons/l10n_pe/models/account_move.py

```diff
@@ -32,6 +43,17 @@ def _inverse_l10n_latam_document_number(self):
             number = rec.l10n_latam_document_number.split("-")
             rec.l10n_latam_document_number = "%s-%s" % (number[0], number[1].zfill(8))
 
+    @api.depends('l10n_latam_available_document_type_ids')
+    def _compute_l10n_latam_document_type(self):
+        result = super()._compute_l10n_latam_document_type()
+        for move in self.filtered(
+                lambda move: move.state == 'draft' and
+                move.company_id.country_id.code == 'PE' and
+                move.partner_id.l10n_latam_identification_type_id.l10n_pe_vat_code != '6' and
+                move.journal_id.type == 'sale'):
+            move.l10n_latam_document_type_id = self.env.ref('l10n_pe.document_type02')
```

Comment:
Just a small remark: this is more than just a default value, we will crush the previous value of `l10n_latam_document_type_id` as soon as `l10n_latam_available_document_type_ids` changes (which is a computed field too, triggered by `'journal_id', 'partner_id', 'company_id', 'move_type', 'debit_origin_id'`). I guess it's ok :)